standards/SPIFFE_Federation.md

```diff
@@ -0,0 +1,316 @@
+
+### 7.1. Distribution of Endpoint Parameters
+The configuration parameters for a federation relationship including trust domain name, endpoint URL, and profile are themselves highly sensitive to tampering. Compromise of the configuration of a federation relationship can weaken or completely break security guarantees implied by an uncompromised SPIFFE implementation.
+
+Some examples:
+* tampering with the trust domain name allows the party that controls the corresponding bundle endpoint to impersonate arbitrary trust domains
+* tampering with the endpoint URL, especially when used in conjunction with the `https_web` profile allows an attacker to issue fraudulent keys and impersonate any identity in the corresponding trust domain
+* tampering with the endpoint profile can alter the security guarantees of the federation, for example substituting `https_spiffe` with `https_web`. If your threat model includes compromise of Web PKI (please additionally see the [Network Traffic Interception](#752-network-traffic-interception) section below), this could be considered a significant downgrade in security posture
+
+Therefore, control plane administrators must take care to source these parameters securely and input them securely. Bundle endpoint configurations can be sourced using a wide variety of methods, including, but not limited to, email, an HTTPS-protected website, a company-internal wiki, etc. Regardless of the specific method used for initial distribution of the endpoint configuration, the distribution method needs to be resistant to in-flight tampering, unauthorized modification at-rest, and malicious impersonation. For example, email is generally not resilient to tampering nor to impersonation (i.e. "spoofed" email).
+
+### 7.2. Explicitly-defined Endpoint Parameters
+Each SPIFFE Federation relationship is configured with the following parameters at minimum:
+* Trust domain name
+* Endpoint URL
+* Endpoint Profile
+
+It is important that these three parameters are configured explicitly, the values cannot be securely inferred from each other.
+
+For example, one might be tempted to infer the SPIFFE trust domain name from the host portion of the Endpoint URL. Firstly, we emphasize that a SPIFFE trust domain name need not be a registered DNS name.  Secondly, inferring this as a default is dangerous because it would allow anyone who can get a file served from a particular DNS name to assert trust roots for the SPIFFE trust domain of the same name. 
+
+Imagine a web hosting company called `spiffyhost.com`, that allows a customer, Alice, to serve web content at URLs like `https://spiffyhost.com/alice/<filename>`, and further, that `spiffyhost.com` operates an API secured by SPIFFE Federation with the SPIFFE trust domain name `spiffyhost.com`.  Imagine Alice sets up SPIFFE Federation with Bob, who is also a customer of `spiffyhost.com`, and Alice chooses to serve her trust bundle from `https://spiffyhost.com/alice/spiffe-bundle`.
```

Comment:
Is this domain name owned by SPIFFE? It's a real domain name....

Reply:
Pretty sure it isn't.  I think it was unregistered when I made up this example.  Should we rewrite with `example.com`?

Reply:
I think it's a good idea to stick with the reserved names. All example subdomains are also reserved, so we could do something like `host-my-page.example.com`. Choosing a name that carries with it the notion that folks may self-serve content there will help drive the example home.